dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -26,7 +26,7 @@ debs/{{ deb }}{{' '}}
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-COPY ["start.sh", "orchagent.sh", "/usr/bin/"]
+COPY ["start.sh", "orchagent.sh", "swssconfig.sh", "/usr/bin/"]
```

Comment:
I see yours as well. This has the potential to get confusing in the future. I'll consider submitting a PR that changes the names of all of these new process start scripts at once.